.gitmodules

```diff
@@ -1,6 +1,9 @@
 [submodule "tests/tests"]
 	path = tests/tests
-	url = https://github.com/ethereum/tests.git 
+	url = https://github.com/ethereum/tests.git
+	shallow = true
 [submodule "core-contracts"]
 	path = core-contracts
 	url = https://github.com/0xPolygon/core-contracts
+	shallow = true
+	branch = dev-ethernal
```

Comment:
This is not the branch we are running for release, but it is `feat-polybft-release` instead (https://github.com/0xPolygon/core-contracts/tree/feat-polybft-release).

Reply:
Just checked and wanted to mention the same)

Reply:
done

Reply:
@Stefan-Ethernal @b00ris